src/System.Runtime.WindowsRuntime.UI.Xaml/pkg/build/net45/System.Runtime.WindowsRuntime.UI.Xaml.targets

```diff
@@ -19,4 +19,13 @@
       </ItemGroup>
     </When>
   </Choose>
+  <!-- when this package is referenced as a nuget reference the binding redirect is still present, add a target to remove it -->
+  <Target Name="_RemoveWindowsRuntimeSuggestedRedirect" 
```

Comment:
This should have a different name so that both this target and the one in System.Runtime.WindowsRuntime can work together without overriding each other.

Reply:
Great point.  Fixed